joomla/joomla-cms
# [4.0] Combining SQL update scripts

This PR merges the update scripts into fewer files. It combines the different SQLs partially by tables, partially by topic. It also orders them to first do modifications, then deletions, then insertions and then updates.

I understand that this is difficult to test. I've only done the MySQL queries so far, since I first wanted to ask if we should do it this way or not. This is "fun" work and if such a big PR is not wanted or if we shouldn't reorder the queries, then I would want to clear that up first before I go through all postgres queries as well.

### How to test
1. Install Joomla 3.10 either on MySQL or Postgres
2. In the Joomla update component, set this server as the source: https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/29374/downloads/34477/pr_list.xml
3. Search for an update and install the update to 4.0
4. Go to the database checker and see if it throws any errors.

If not and everything else is working as expected, you are good to go.
